docs/notebooks/hydrotel.ipynb

```diff
@@ -0,0 +1,326 @@
+        "\n",
+        "<div class=\"alert alert-info\"> <b>INFO</b>\n",
+        "`xhydro` does not distribute the HYDROTEL executable. If you want to use this module, you will have to get access to it first.\n",
+        "</div>\n",
+        "\n",
+        "### Initialising the model\n",
+        "`xhydro` does not prepare the project directory itself, which should be done beforehand. What the class does, when initiating a new instance of `xhydro.modelling.Hydrotel`, is allow modifying the entries located in the three main configuration files: `projet.csv`, `simulation.csv`, and `output.csv`. At minimum, when this class is initialised, the project folder will have to already exist and either 'default_options' will have to be True or SIMULATION COURANTE (current simulation) must be specified as a keyword argument in `project_options`.\n",
+        "\n",
+        "The arguments of the class are:\n",
+        "\n",
+        "- `project`: *str* or *os.Pathlike*. The full path to the project directory.\n",
+        "- `default_options`: *bool*. Whether to use default configuration options taken from `xhydro/modelling/data/hydrotel_defaults.yml`, or read them from the files on disk.\n",
+        "- `project_options`, `simulation_options`, `output_options`: *dict*. Configuration options to overwrite with new information.\n",
+        "\n",
+        "At any time after initialising the class, `Hydrotel.update_options()` can be called to update the three configuration files. When called, this function will overwrite the CSV files written on disk."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "25970397-024f-428f-92b6-1675a80e0161",
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "from pathlib import Path\n",
+        "from xhydro.modelling import Hydrotel\n",
+        "import xhydro as xh\n",
+        "\n",
+        "# Folder where to put the data\n",
+        "project_folder = Path().absolute() / \"_data\"\n",
+        "project_name = \"example_hydrotel\""
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "fc3a8459-c647-41c4-9388-e896c99af7ff",
+      "metadata": {
+        "editable": true,
+        "nbsphinx": "hidden",
+        "slideshow": {
+          "slide_type": ""
+        },
+        "tags": [
+          "hide_code"
+        ]
+      },
+      "outputs": [],
+      "source": [
+        "# This is a hidden cell. We'll create a fake Hydrotel directory for the purpose of this example.\n",
+        "from xclim.testing.helpers import test_timeseries as timeseries\n",
+        "import xhydro.testing.utils\n",
+        "import numpy as np\n",
+        "\n",
+        "# Fake meteorological data\n",
```

Comment:
This looks like the same object as found in the (now) conftest. Maybe it would be useful in a `xhydro.testing.helpers` module?

Reply:
True. I'll merge the functionality with `fake_hydrotel_project`.